uber/templates/summary/index.html

```diff
@@ -13,7 +13,9 @@
     <b>Attendees who paid and didn't show up:</b> {{ counts.noshows.paid }} <br/>
 {% endif %}
 <b>Number of remaining badges for sale (before event is sold out):</b> {{ c.REMAINING_BADGES }} <br/>
-<b>Number of attendees who've donated at least ${{ c.SUPPORTER_LEVEL }}:</b> {{ counts.supporter_level }}
+{% for amount, count in counts.donation_tiers.items() %}
+  <b>Number of attendees who've donated at least ${{ amount }}:</b> {{ count }} <br>
```

Comment:
Super minor nitpick here -- the wording implies that this will cumulatively count attendees, i.e., attendees who donated $50 would be included in both the $25 and $50 tiers. However, from the code it looks like it's not cumulative. It'd be nice if the wording was clearer about it.